pkg/types/telemetry.go

```diff
@@ -6,10 +6,19 @@ import (
 	"github.com/smartcontractkit/libocr/commontypes"
 )
 
-type Telemetry interface {
+type TelemetryService interface {
 	Send(ctx context.Context, network string, chainID string, contractID string, telemetryType string, payload []byte) error
 }
 
+type TelemetryClientEndpoint interface {
+	SendLog(ctx context.Context, log []byte) error
+}
+
+type TelemetryClient interface {
+	TelemetryService
+	NewEndpoint(nework string, chainID string, contractID string, telemetryType string) (TelemetryClientEndpoint, error)
```

Comment:
I think we want to include ctx and error by default on everything new. Any reason not to here?
```suggestion
	NewEndpoint(ctx context.Context, nework string, chainID string, contractID string, telemetryType string) (TelemetryClientEndpoint, error)
```

Reply:
No, this was just because I renamed the method from GenMonitoringEndpoint -- definitely agree we want to pass in a context here :)